### DISCUSSION TITLE
How can I always assign the same IP from the WiFi range extender to the client via MAC

### DISCUSSION THREAD

**Discussion by MCSE-2000** [arendst/Tasmota#21190] [Support request]
How can I always assign the same IP to the client using the MAC in the DHCP of the Wi-Fi range extender? I have compiled a ShellyPlugS with Wi-Fi range extender. It works, but the client (APsystems EZ-M1) is always assigned a different IP every day. I cannot enter a static IP on the EZ-M1 client.

**Comment by sfromis** [comment]
Did you try *not* using the simpler NAPT mode?

More generally, for "normal Tasmota" operation via MQTT, the IP address should really not matter.

**Comment by MCSE-2000** [comment]
I use routing (not NAPT). The APSYSTEMS EZ1-M inverter cannot use MQTT. It has an HTTP API on port 8050, but it gets a different IP from Tasmota every day and I have to search for the EZ1-M in the network.

**Comment by Jason2866** [comment]
No fix will be done to the deprecated Range Extender version. Migration of the actual not working Range Extender driver for Tasmota Arduino Core 3.0.0 has just started.
